tests/functional/spec/conditional_dates.js

```diff
@@ -3,7 +3,7 @@ const DatesPage = require('../generated_pages/conditional_dates/date-block.page'
 const DatesConfirmationPage = require('../generated_pages/conditional_dates/date-value-test.page');
 const SummaryPage = require('../generated_pages/conditional_dates/summary.page');
 
-describe('Piped Dates', function () {
+describe.skip('Piped Dates', function () {
```

Comment:
👀

Reply:
Is a test we want to keep but won’t currently work